### DISCUSSION TITLE
Question: obtain mouse position in world coordinates

### DISCUSSION THREAD

**Discussion by iulianR** [doonv/bevy_smooth_pixel_camera#9] [Q&A] [answered]
Hello,

Code like this one for obtaining the mouse position in world coordinates doesn't look like it would work anymore.
https://bevy-cheatbook.github.io/cookbook/cursor2world.html

Any hint or API of how it can be done with this camera?

Thank you!

**Comment by doonv** [comment, marked as answer]
You need to convert the cursor position from being a cursor position on the window, to a cursor position on the `PixelCamera`'s viewport.

The way you do this depends on what `ViewportSize` you use (I will probably implement a function for this in the future). But for `ViewportSize::PixelFixed` you can just divide the cursor position by the `value` in `PixelFixed(value)`
